AllReadyApp/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+run_cmd="tail -f /dev/null"
+until dotnet ef database update; do
+>&2 echo "SQL Server is starting up"
+sleep 1
+done
+
```

Comment:
This file seems to only run the migration and does not actually start the app. i.e. `dotnet AllReady.dll` is nowhere to be seen...